go/cmd/vtctldclient/command/vreplication/lookupvindex/lookupvindex.go

```diff
@@ -0,0 +1,285 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lookupvindex
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/maps"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+	"vitess.io/vitess/go/cmd/vtctldclient/command/vreplication/common"
+	"vitess.io/vitess/go/vt/sqlparser"
+
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	vschemapb "vitess.io/vitess/go/vt/proto/vschema"
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+	topoprotopb "vitess.io/vitess/go/vt/topo/topoproto"
+)
+
+var (
+	tabletTypesDefault = []topodatapb.TabletType{
+		topodatapb.TabletType_REPLICA,
+		topodatapb.TabletType_PRIMARY,
+	}
+
+	baseOptions = struct {
+		// This is where the vindex will be created.
+		Keyspace string
+		// This will come from the name of the vindex target table
+		// in the provided spec with a static suffix of `_vdx` added.
+		Workflow string
+		Vindex   *vschemapb.Keyspace
+		// This is where the vindex target table and VReplicaiton
+		// workflow will be created.
+		TargetKeyspace string
+	}{}
+
+	// base is the base command for all actions related to Lookup Vindexes.
+	base = &cobra.Command{
+		Use:   "LookupVindex --keyspace <keyspace> [command] [<vindex spec>]",
+		Short: "Perform commands related to creating, backfilling, and externalizing Lookup Vindexes using VReplication workflows.",
+		Long: `LookupVindex commands: create, show, externalize, and cancel.
```

Comment:
this is redundant (`--help` for a command with children lists its available subcommands)

Reply:
unless you have (sorry) longer help text than feels appropriate for `Short` then yeah

the behavior is `Short` is what gets shown in `vtctldclient --help` next to the `LookupVindex` command, and for `vtctldclient help LookupVindex` it's basically "Long if set, Short otherwise"

Reply:
I went ahead and remove them for all the VReplication commands.